Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.yml

```diff
@@ -42,6 +48,12 @@ configuration:
   - Bad
   additionalinfo: Indicators from this integration instance will be marked with this reputation.
   required: false
+- additionalinfo: 'Improve the feed performance by removing the rawJSON from the indicators. Note: A loss of data may occur if selecting this checkbox.'
```

Comment:
```suggestion
- additionalinfo: 'Should be used in case of performance issues leading to timeouts. Note: When selecting this checkbox, indicator mapping cannot be performed.'
```